src/lib/input/input-container.scss

```diff
@@ -30,3 +30,9 @@ md-input-container {
     }
   }
 }
+
+// Prevents the prefix and suffix from stretching together with the container.
+.md-input-prefix, .md-input-suffix {
+  width: 0.1px;
```

Comment:
Not sure, IIRC @hansl implemented it like this initially, but I don't know what the reasoning is.